<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminpanel</class>
 <widget class="QWidget" name="adminpanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminPanel</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Downloads/4360835.png</normaloff>../Downloads/4360835.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(23, 45, 71);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Admin panel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EnterChange">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/images/1661330481_17-kartinkin-net-p-fon-dlya-avtorizatsii-krasivo-17.png);</string>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/images/1661330481_17-kartinkin-net-p-fon-dlya-avtorizatsii-krasivo-17.png);</string>
   </property>
   <property name="text">
    <string>Exit to menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Users</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Level access</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
